haas/model.py

```diff
@@ -531,19 +531,18 @@ class NetworkingAction(db.Model):
     # * 'revert_port' detaches the port from all networks.
     type = db.Column(db.String, nullable=False)
 
-    nic_id = db.Column(db.ForeignKey('nic.id'), nullable=False)
     new_network_id = db.Column(db.ForeignKey('network.id'), nullable=True)
 
+    # The port affected by the action. for 'revert_port'
```

Comment:
`for 'revert_port'` isn't really very useful here -- it's still getting used for all of the other operations. Instead, I might mirror the comment you removed, saying something like "the port affected by the action. For operations that conceputally operate on nics, this is the port they are attached to."

The fact that the reason for the change has to do with revert_port is something that makes more sense in a commit message -- the reader of the code isn't necessarily interested in a specific point in history other that the present.